src/main/kotlin/the/coding/force/exploring_kotlin_coroutines/service/coroutine/ReadDataServiceCoroutine.kt

```diff
@@ -8,12 +8,32 @@ import the.coding.force.exploring_kotlin_coroutines.exception.DataNotFoundExcept
 import the.coding.force.exploring_kotlin_coroutines.repository.DataRepository
 import the.coding.force.exploring_kotlin_coroutines.response.ReadDataResponse
 
+/**
+ * This Service has the suspend method `read` inside coroutine context to get a `ReadDataResponse`
+ *
+ * @author Lucas Terra
+ * @param dataRepository repository from DataEntity
+ * @property read suspend method to get a `ReadDataResponse` in coroutine context
+ */
```

Comment:
```suggestion
/**
 * @author Lucas Terra
 * @param dataRepository repository from DataEntity
 * @property read suspend method to get a `ReadDataResponse` in coroutine context
 */
```

É mais uma opinião, não é uma regra. 
Acho esse comentário desnecessário porque é fácil de ler o código e interpretar isso.